src/GovUk.Education.ExploreEducationStatistics.Events/ReleaseVersionPublishedEvent.cs

```diff
@@ -40,18 +41,48 @@ public record EventPayload
         public required Guid PublicationId { get; init; }
         public required string PublicationSlug { get; init; }
         public required Guid PublicationLatestPublishedReleaseVersionId { get; init; }    
+        public Guid? PreviousLatestReleaseId { get; init; }
     }
     public EventPayload Payload { get; }
     
     public EventGridEvent ToEventGridEvent() => new(Subject, EventType, DataVersion, Payload);
 
     public record PublishedReleaseVersionInfo
     {
+        /// <summary>
+        /// Newly published release version id
+        /// </summary>
         public Guid ReleaseVersionId { get; init; }
+    
+        /// <summary>
+        /// The Release Id for the newly published release version
+        /// </summary>
         public Guid ReleaseId {get;init;}
+        
+        /// <summary>
+        /// The release slug for the newly published release version
+        /// </summary>
         public string ReleaseSlug { get; init; } = string.Empty;
+        
+        /// <summary>
+        /// The publication id for the newly published release version
+        /// </summary>
         public Guid PublicationId { get; init; }
+        
+        /// <summary>
+        /// The publication slug for the newly published release version
+        /// </summary>
         public string PublicationSlug { get; init; } = string.Empty;
+        
+        /// <summary>
+        /// The release version that has been published may not be the latest.
+        /// This property contains the latest published release version id.
+        /// </summary>
         public Guid PublicationLatestPublishedReleaseVersionId { get; init; }
+        
+        /// <summary>
+        /// The Release Id of the previous "latest release version"
+        /// </summary>
+        public Guid? PreviousLatestReleaseId { get; init; }
```

Comment:
I think we could name this PreviousLatestPublishedReleaseId to remove any ambiguity over whether its the latest release or the latest published release.

It's then consistent with the naming of `PublicationLatestPublishedReleaseVersionId`. Alternatively we could go the other way and shorten that one. I think it depends if you think the fact it's in a type that already contains the name 'Published' gives it away.

You don't make this change but if you agree I can make the change when the next PR comes along.